apps/labs/next.config.js

```diff
@@ -30,18 +30,78 @@ const nextConfig = {
       },
     ];
   },
-  async rewrites() {
+  async redirects() {
     return [
       {
-        source: '/plausible/js/script.js',
-        destination: 'https://plausible.io/js/script.js',
+        source: '/about',
+        destination: '/',
+        permanent: true,
       },
       {
-        source: '/plausible/api/event',
-        destination: 'https://plausible.io/api/event',
+        source: '/blog/:yyyy(\\d{4})/:mm(\\d{2})/:slug',
+        destination: '/blog/:slug?yyyy=:yyyy&mm=:mm',
+        permanent: true,
       },
     ];
   },
+  async rewrites() {
+    // The conventions of this return value are described in the Next.js docs:
+    // https://nextjs.org/docs/api-reference/next.config.js/rewrites
+    return {
+      afterFiles: [
+        // These rewrites are checked after pages/public files are checked but
+        // before dynamic routes.
+
+        // Proxying Plausible through Vercel
+        // https://plausible.io/docs/proxy/guides/vercel
+        {
+          source: '/plausible/js/script.js',
+          destination: 'https://plausible.io/js/script.js',
+        },
+        {
+          source: '/plausible/api/event',
+          destination: 'https://plausible.io/api/event',
+        },
+      ],
+      fallback: [
+        // These rewrites are checked after both pages/public files and dynamic
+        // routes are checked.
+        {
+          source: '/blog/:slug',
```

Comment:
For some reason this works locally (localhost:4200) but not on vercel.app.

Here's what happens locally:

- I visit the URL to a page that hasn't yet been migrated, such as `blog/2019/05/labs-update-april-highlights`.
- Local server follows redirect rule above and redirects the user's browser to `/blog/labs-update-april-highlights?yyyy=2019&mm=05.
- There is no blog post currently at that URL on localhost, so local server then proxies the request as expected, and renders the `labs-update-april-highlights` blog post from the old site, in other words it shows the exact same page you would get if you get the page from Netlify at this URL: https://quansight-labs.netlify.app/2019/05/blog/labs-update-april-highlights/. The URL in the browser, of course, stays the same.

What happens in production:

- I visit the URL, for example: https://quansight-labs-staging-git-why-doesnt-this-work-quansight.vercel.app/blog/2019/05/labs-update-april-highlights
- It gets redirected as expected.
- But, instead of proxying the old blog post from Netlify, we get the old Netlify 404.

I don't know why this works locally but fails on Vercel.app.